analysis/src/SignatureHelp.ml

```diff
@@ -152,159 +152,206 @@ let docsForLabel typeExpr ~file ~package ~supportsMarkdownLinks =
   typeString :: typeDefinitions |> String.concat "\n"
 
 let signatureHelp ~path ~pos ~currentFile ~debug =
-  let posBeforeCursor = Pos.posBeforeCursor pos in
```

Comment:
Absolutely terrible display of the diff in GitHub. I've added two optional unwrappings which messes with the indentation, which messes up the diff. Haven't really changed that much.